R/pkg/R/generics.R

```diff
@@ -1168,3 +1168,7 @@ setGeneric("kmeans")
 #' @rdname fitted
 #' @export
 setGeneric("fitted")
+
+#' @rdname naiveBayes
+#' @export
+setGeneric("naiveBayes", function(formula, data, ...) { standardGeneric("naiveBayes") })
```

Comment:
Users need `e1071::naiveBayes` to call that. Both `naiveBayes` are implemented as S3 generic functions and the Spark one shares the same first argument type (`formula`) with the one in e1071. So I don't think we can avoid shadowing the method. I tried different loading orders and confirmed that both can be used with namespace prefixes.